.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "library"]
+	path = library
+	url = https://github.com/h5p/h5p-php-library.git
```

Comment:
I noticed this change in the `2015-dev` branch and rebased on it. When and how is this chance supposed to become part of the `master` branch?